spring-boot-project/spring-boot-dependencies/build.gradle

```diff
@@ -1416,13 +1423,20 @@ bom {
 			]
 		}
 	}
-	library("Rabbit AMQP Client", "5.13.0") {
+	library("RabbitMQ AMQP Client", "5.13.0") {
```

Comment:
This will result in a breaking change to the version property. It's `rabbit-amqp-client.version` at the moment and this will change it to `rabbitmq-amqp-client.version`. I'd prefer to leave it unchanged, but if we want to change it, it should be done in a separate issue.

Reply:
That's fine; I just wanted to make it consistent; will revert.